operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -613,7 +626,15 @@ public boolean certsRemoved() {
      * @return Whether the certificate was renewed.
      */
     public boolean certRenewed() {
-        return renewalType == RenewalType.RENEW_CERT || renewalType == RenewalType.REPLACE_KEY;
+        return renewalType == RenewalType.RENEW_CERT || renewalType == RenewalType.REPLACE_KEY || certInfoChanged();
```

Comment:
Why do we use `renewalType == RenewalType.RENEW_CERT` and `renewalType == RenewalType.REPLACE_KEY` and not `renewalType == RenewalType. REGENERATED_CERT_INFO `

Reply:
I am not sure what you mean by this. `renewalType = RenewalType.RENEW_CERT `and `renewalType = RenewalType.REPLACE_KEY` are set on another places so changing this condition just to the `renewalType == RenewalType. REGENERATED_CERT_INFO` would break some things, would not it?